client/src/js/services/receipts/ReceiptService.js

```diff
@@ -21,14 +19,19 @@ function ReceiptService($http, util) {
   /**
    * Fetch invoice report data from /reports/invoices/:uuid
    *
-   * @param {String} uuid      Target invoice UUID to report on
-   * @param {Object} options   Configuration options for the server generated
+   * @params {String} uuid      Target invoice UUID to report on
+   * @params {Object} options   Configuration options for the server generated
```

Comment:
The correct format for this tag is [@param](http://usejsdoc.org/tags-param.html), not `@params`.